src/app/components/cds/inputs/TextArea.js

```diff
@@ -1,20 +1,36 @@
 // DESIGNS: https://www.figma.com/file/rnSPSHDgFncxjXsZQuEVKd/Design-System?type=design&node-id=3606-26274&mode=design&t=ZVq51pKdIKdWZicO-4
 import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import TextField from './TextField';
 
 const TextArea = React.forwardRef(({
-  autoGrow = true,
+  autoGrow,
+  maxHeight,
+  minHeight,
   ...inputProps
 }, ref) => {
   const [height, setHeight] = useState('auto');
 
   const handleChange = (event) => {
-    if (autoGrow) {
-      setHeight(`${event.target.scrollHeight}px`);
+    const newHeight = `${event.target.scrollHeight}px`;
+    if (autoGrow && event.target.scrollHeight >= minHeight && (!maxHeight || event.target.scrollHeight <= maxHeight)) {
+      setHeight(newHeight);
     }
   };
 
-  return <TextField textArea ref={ref} {...inputProps} style={{ height }} onChange={handleChange} />;
+  return <TextField textArea autoGrow ref={ref} {...inputProps} style={{ height }} onChange={handleChange} />;
 });
 
+TextArea.defaultProps = {
+  autoGrow: true,
+  maxHeight: null,
+  minHeight: 65,
```

Comment:
I'm not sure this is the best way to approach this. The reason is, the TextField css field css file has a min-height set for text area at `min-height: 3.5em;` and I think its a bad idea to have two different ways of setting this. Some suggestions are:
- in the js look at the css value, I think its `getComputedStyle`
- go down to 1 set in the js, and make it required so its clear to developers whats going on
- other? any comment here @amoedoamorim ?

Reply:
This might be coming a bit late, but I wonder if this could be solved with an almost css-only approach in which autoGrow would mean applying either styles (with or without maxHeight). Wouldn't it be cleaner? Am I missing something here?